src/Peering/Peering.sln

```diff
@@ -21,6 +21,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ScenarioTest.ResourceManage
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestFx", "..\..\tools\TestFx\TestFx.csproj", "{BC80A1D0-FFA4-43D9-AA74-799F5CB54B58}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Resources", "..\Resources\Resources\Resources.csproj", "{E1F5201D-6067-430E-B303-4E367652991B}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ResourceManager", "..\Resources\ResourceManager\ResourceManager.csproj", "{8058D403-06E3-4BED-8924-D166CE303961}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tags", "..\Resources\Tags\Tags.csproj", "{6BD4D521-DAFB-472B-A803-81F053AB1396}"
+EndProject
```

Comment:
These projects should not be necessary for testing (which I'm assuming why they were added). For tests, you should be using the functions from this script: https://github.com/Azure/azure-powershell/blob/master/tools/ScenarioTest.ResourceManager/AzureRM.Resources.ps1

Reply:
Cormac added these yesterday for commit a02b13a. Let me know how to proceed.

Reply:
Talked to Cormac. This is the 'quick solution' to making your solution file build and run tests. The preferred way it what I mentioned above, using the functions in `AzureRM.Resources.ps1`. However, that will likely cause you to need to re-record your tests.